easybuild/easyconfigs/p/PRRTE/PRRTE-3.0.5-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PRRTE'
+version = '3.0.5'
+
+homepage = 'https://docs.prrte.org/'
+description = """PRRTE is the PMIx Reference RunTime Environment"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openpmix/prrte/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.bz2']
+checksums = ['75ce732b02f3bc7eff5e51b81469e4373f1effc6a42d8445e2935d3670e58c8e']
+
+builddependencies = [('binutils', '2.42')]
+
+dependencies = [
+    ('libevent', '2.1.12'),
+    ('hwloc', '2.10.0'),
+    ('PMIx', '5.0.2'),
+]
+
+configopts = ' --with-libevent=$EBROOTLIBEVENT'
+configopts += ' --with-hwloc=$EBROOTHWLOC --with-pmix=$EBROOTPMIX'
+
+buildopts = 'V=1'
+
+sanity_check_paths = {
+    'files': ['bin/prte', 'bin/prte_info', 'bin/prterun', 'bin/prun', 'bin/pterm'],
```

Comment:
@bartoldeman Could you add sanity check commands for these binaries?

Reply:
Done